dynamic/Dockerfile

```diff
@@ -92,14 +93,29 @@ RUN ( \
         && chmod 755 /usr/bin/runc \
     ) || true
 
+# On s390x install docker ce, runc and containerd from docker static s390x build
+RUN ( \
+        [[ "${TARGETPLATFORM}" = 'linux/s390x' ]] \
+        && microdnf install --nodocs tar gzip wget \
+        && curl -L --fail --show-error --silent https://download.docker.com/linux/static/stable/s390x/docker-18.06.3-ce.tgz -o /tmp/docker-18.06.3-ce.tgz \
```

Comment:
would be nice to first change to `/tmp` before downloading. The file is correctly downloaded there, but then the `tar -xzf` is being executed in the working directory. Alternatively let it extract to `/tmp`.
Now its uncertain where the `docker` directory ends up and whether it might conflict with anything else there.

Reply:
done